### PR TITLE
Gt 421 us kgp share link

### PR DESCRIPTION
**Pull request by ryancarlson**
- Translates code from kgp-us to kgp for "Know God Personally"
- Adds page number to URLS for i > 0
- But no page numbers for US KGP

### CODE REVIEW

File: library/tract-renderer/src/main/java/org/cru/godtools/tract/model/Manifest.java [hunk 1]
```diff
@@ -114,6 +114,11 @@ public String getCode() {
     }
 
     @NonNull
+    public String getCodeForShareActivity() {
+        return mCode.equals("kgp-us") ? "kgp" : mCode;
```
Comment (frett) [inline library/tract-renderer/src/main/java/org/cru/godtools/tract/model/Manifest.java]: I think I want to keep this custom sharing override logic localized to the TractActivity for now.

**Comment by ryancarlson** [comment]
@frett made the change you suggested. are these still the travis failures we had been seeing?

**Comment by frett** [comment]
part of the travis log says checkstyle rule errors

**Comment by frett** [comment]
```
[ant:checkstyle] [ERROR] /Users/frett/development/android/godtools-android/library/tract-renderer/src/main/java/org/cru/godtools/tract/activity/TractActivity.java:601:35: String literal expressions should be on the left side of an equals comparison. [EqualsAvoidNull]
```

**Comment by ryancarlson** [comment]
thanks!

**Pull request merged**